pkg/util/errors.go

```diff
@@ -20,29 +20,40 @@ import "bytes"
 
 // Errors implements error and contains all config errors
 // This is returned by Config.Validate
-type Errors struct {
-	errors []error
-}
+type Errors []error
```

Comment:
I couldn't think of anything valid to add to this struct later, and doing this significantly cleaned up code like `Flatten` imo, which was very useful for the log collection

Reply:
👍 there is also `k8s.io/apimachinery/pkg/util/errors.ToAggregate` otherwise